### PR TITLE
[Robot] Verify accounts and contacts checkboxes can retain values on save

### PR DESCRIPTION
**Pull request by rhakeshj**
Verify accounts and contacts setting checkboxes can retain values on save
Removed unstable flags for the following robot files

- Courses 
- Course Connections
- Backfill
- Hierarchy Settings
- Relationships auto
- Relationships reciprocal

**Robot WI** - W - 8620850

**New/Existing Test** - Both

To run the test case, run this from your CCI:
cci task run robot -o suites robot/EDA/tests/browser/settings/accounts_contacts.robot

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: robot/EDA/tests/browser/settings/accounts_contacts.robot [hunk 1]
```diff
@@ -59,4 +59,48 @@ Verify disable preferred phone enforcement shows and hides as expected
...
+Validate all checkboxes can retain value on save in accounts and contacts settings
+    [Documentation]         Validates that all checkbox fields in accounts and contacts settings
+    ...                     page retains value after clicking on save button. The status of checkbox
+    ...                     is true if checked and false if unchecked
+    [tags]                  unstable        W-8620850       rbt:high
+    Click action button on EDA settings page    Edit
+    Set checkbox value
+    ...                 Disable Preferred Email Enforcement=true
+    ...                 Enable Enhanced Preferred Phone Functionality=true
+    ...                 Disable Preferred Phone Enforcement=true
```
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/accounts_contacts.robot:73]: If this test case fails in between the execution, would the point of failure affect any other test cases in the future ? Just checking if adding any teardown steps (either now or at a later point) would help with inconsistencies.

**Review by Safiyab — APPROVED** [review]
Looks good.

**Review by smparedes — APPROVED** [review]
@rhakeshj approved!

**Review by spinkelman — APPROVED** [review]
After Functional Review this is Approved.

**Pull request merged**